stdlib/compat.ncl

```diff
@@ -9,6 +9,14 @@ compat | doc "Nix compatibility layer. This library should not be used by Nickel
     then a ++ b
     else a + b,
 
+  # The update operator of Nix `//`. It's a "general form" of the `record.update` of Nickel.
+  # TODO: May be intresting to be addapted and integrated the the actual Nickel stdlib.
+  update_from_rec = fun r1 r2 =>
```

Comment:
Just a suggestion, but for the naming: what about `update_all`? Or at least `update_from_record`? `rec` is a bit ambiguous in Nickel, as it also stands for _recursive_.

Reply:
I like the `update_all`